graviti/manager/draft.py

```diff
@@ -300,6 +301,9 @@ def create(
             branch=branch,
             description=description,
         )
+
+        if getattr(head, "name", None) == branch and head.commit_id != response["parent_commit_id"]:
```

Comment:
Is `head` must be `Branch` here? do we need to use `getattr`?

Reply:
head could be commit.
```
dataset.checkout("commit_id")
dataset.drafts.create("draft-1", branch="main")
```